app/controllers/nsm/additional_fees_controller.rb

```diff
@@ -38,12 +38,21 @@ def show
 
     def edit
       authorize(claim, :edit?)
-      rows = BaseViewModel.build(:additional_fees_summary, claim).rows
-      item = rows.detect do |model|
-        model.type == params[:id].to_sym
-      end
 
-      render locals: { claim:, item:, }
+      item = BaseViewModel.build(:additional_fee, claim)
```

Comment:
There will likely be other kinds of fees in the future so we can't rely on this always being the right one hence the `detect` check to get the right one